lib/http.js

```diff
@@ -60,16 +58,7 @@ export async function addRoutes({app, service} = {}) {
       try {
         const {config} = req.serviceObject;
         const {credential, options} = req.body;
```

Comment:
Needs JSON schema changes as well -- that only permits `mandatoryPointers` for now.

Reply:
Added it here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/130/commits/b483bf69420ef229b1a70e1e73b8dedbfa115226

Do we want an RegExp associated with it? I know that a pointer that doesn't start with `/` seems to cause it to throw.

Reply:
No, we don't need to go that far.